mathesar-foundation/mathesar-website
# Update GitHub Pages gem to fix issue with Ruby versions >= 3.3.0

The previous version of the GitHub pages gem, v227, had an issue with the latest version of Ruby (any greater than 3.3.x, or any version using ruby/logger 1.4.3 or greater. See the issue here: https://github.com/jekyll/jekyll/pull/9392). Updating this gem to v232 updates Jekyll to > 3.9.4, where the logger compatibility issue is fixed.

This is a more dependable solution than some alternatives:

- Downgrade my personal Ruby version and enforce a specific version < 3.3.0
- Updating the jekyll gem directly with `bundle update jekyll`

## Testing instructions

1. Checkout this PR and `bundle install`.
2. Run the site locally with `bundle exec jekyll serve --port 4000 --livereload` and confirm it matches the production site
3. **Bonus, optional:** Checkout main, update your Ruby version to something > 3.3.0 (Perhaps with `rvm install ruby --latest` or `rbenv install 3.3.1`) and verify that the site fails to run locally with an error message mentioning the `@level_override[Fiber.current]` configuration.
